src/nvme/fabrics.c

```diff
@@ -459,8 +463,11 @@ static int build_options(nvme_host_t h, nvme_ctrl_t c, char **argstr)
 			 nvme_ctrl_get_trsvcid(c)) ||
 	    (hostnqn && add_argument(argstr, "hostnqn", hostnqn)) ||
 	    (hostid && add_argument(argstr, "hostid", hostid)) ||
+	    (hostsymname && add_argument(argstr, "hostsymname", hostsymname)) ||
```

Comment:
Can you please split this off into several parts?
I would like to see:
a) a patch for adding the 'hostsymname' option
b) a patch for adding explicit registration support (ie command description and tooling) to libnvme/nvme-cli such that users can send / test the commands from userspace
c) a patch for adding the 'register' option to nvme-cli, once the discussion on linux-nvme has been resolved.